content/en/authors/watroba/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+# Display name
+title: Marcin Wątroba
+
+# Username (this should match the folder name)
+authors:
+- watroba
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: PhD Student
```

Comment:
please change the PhD Student to Project's staff role

Reply:
Ok